src/renderer/atlas/atlas.vcxproj

```diff
@@ -12,16 +12,21 @@
   <ItemGroup>
     <ClCompile Include="AtlasEngine.api.cpp" />
     <ClCompile Include="AtlasEngine.r.cpp" />
+    <ClCompile Include="dwrite.cpp" />
     <ClCompile Include="pch.cpp">
       <PrecompiledHeader>Create</PrecompiledHeader>
     </ClCompile>
     <ClCompile Include="AtlasEngine.cpp" />
   </ItemGroup>
   <ItemGroup>
+    <ClInclude Include="dwrite.h" />
     <ClInclude Include="pch.h" />
     <ClInclude Include="AtlasEngine.h" />
   </ItemGroup>
   <ItemGroup>
+    <FxCompile Include="dwrite.hlsl">
+      <ExcludedFromBuild>true</ExcludedFromBuild>

```

Comment:
Wait, why excluded? You don't want it precompiled?

Reply:
No, because it's `#include`d from the `shader.hlsl` file already.